examples/154_ReflectC.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Circle inversion</title>
+    <link rel="stylesheet" href="../build/js/CindyJS.css">
+    <script type="text/javascript" src="../build/js/Cindy.js"></script>
+    <script type="text/javascript">
+var cdy = CindyJS({
+  scripts: "cs*",
+  defaultAppearance: {
+    dimDependent: 0.7,
+    fontFamily: "sans-serif",
+    lineSize: 1,
+    pointSize: 5.0,
+    textsize: 12.0
+  },
+  angleUnit: "°",
+  geometry: [
+    {name: "A", type: "Free", pos: [4.0, 3.2, 0.8], color: [1.0, 0.0, 0.0], labeled: true},
```

Comment:
The free point `A` appears nowhere else in the construction. Perhaps it should be put to better use as the center of the circle `C0`. Also a point should be added for the center of the circle `C3` as that is all that remains when `C2's ` radius is reduced to zero. Of course I found that I had to increase  `C2's ` radius from one to two in order to be able to stretch it at all. Also it might be nice to add a bit more color to associate the objects and their reflections in the circle `C0`.